src/renderer/cursor_renderer/mod.rs

```diff
@@ -422,6 +422,7 @@ impl CursorRenderer {
         if !animating {
             self.previous_editor_mode = current_mode.clone();
         }
+        tracy_plot!("cursor animating", animating as u8 as f64);
```

Comment:
What is the thought behind the double cast?

Reply:
The plots only takes f64, and animating as a boolean, which is not directly convertible to floats

Reply:
Oh. That makes sense, thanks!